pkg/collector/python/init.go

```diff
@@ -240,22 +239,36 @@ func Initialize(paths ...string) error {
 		}
 	}
 
+	if pythonVersion == "2" {
+		PythonHome = pythonHome2
+	} else if pythonVersion == "3" {
+		PythonHome = pythonHome3
+	}
+
+	if runtime.GOOS == "windows" {
+		pythonBinPath = filepath.Join(PythonHome, "python.exe")
+	} else {
+		pythonBinPath = filepath.Join(PythonHome, "bin", "python")
```

Comment:
note: on Linux, this will not necessarily point to the python binary of the python version that's actually enabled on the running Agent, but it will point to the python version that's considered "default" in the current installation.

A bit outside the scope of this PR, should be addressed separately

Reply:
it actually will I think following this block https://github.com/DataDog/datadog-agent/pull/4052/files/8a2fee0966e9a9cd223ea04eb1438197986bc128#diff-ce96e6f2b410b7d5f58d8a244ca0fb62R242 or am I missing something ?

Reply:
@hush-hush unless I'm mistaken, on Linux `PythonHome` will be set to `/opt/datadog-agent/embedded` whether python 2 or python 3 is used. The python binaries are both located in `/opt/datadog-agent/embedded/bin`. So by using `/opt/datadog-agent/embedded/bin/python` here, you're using the python binary of the python version that's considered "default" for this specific package of the Agent.

For Agent 6.x this "default" will remain python 2, but for Agent 7 this will switch to python3, so we'll want to update this before 7.0.

Feel free to either fix this now or create a card for Agent 7 on the trello board :)